install/kubernetes/quick-install.yaml

```diff
@@ -653,7 +698,7 @@ spec:
               key: debug
               name: cilium-config
               optional: true
-        image: "docker.io/cilium/operator-generic:latest"
+        image: cilium/operator-generic:latest
```

Comment:
this is missing the registry `docker.io`

Reply:
done